main.py

```diff
@@ -189,6 +192,125 @@ def updatebyemail():
         else:
             return jsonify(respo='Subscription already exist', )
 
+
+class LoginForm(form.Form):
+    login = fields.TextField(validators=[validators.required()])
+    password = fields.PasswordField(validators=[validators.required()])
+
+    def validate_login(self, field):
+        user = self.get_user()
+
+        if user is None:
+            raise validators.ValidationError('Invalid user')
+
+        if not check_password_hash(user.password, self.password.data):
+            raise validators.ValidationError('Invalid password')
+
+    def get_user(self):
+        return db_session.query(User).filter_by(login=self.login.data).first()
+
+
+class RegistrationForm(form.Form):
+    login = fields.TextField(validators=[validators.required()])
+    email = fields.TextField()
+    password = fields.PasswordField(validators=[validators.required()])
+
+    def validate_login(self, field):
+        if db_session.query(User).filter_by(login=self.login.data).count() > 0:
+            raise validators.ValidationError('Duplicate username')
+
+
+def init_login():
+    login_manager = login.LoginManager()
+    login_manager.init_app(app)
+
+    # Create user loader function
+    @login_manager.user_loader
+    def load_user(user_id):
+        return db_session.query(User).get(user_id)
+
+
+class MyModelView(sqla.ModelView):
```

Comment:
Why "My" (MyModelView, MyAdminIndexView, my_master)? Is that the standard in Flask apps?